app/presenters/email_alert_presenter.rb

```diff
@@ -24,6 +24,12 @@ def body
     end
   end
 
+  def tags
+    {
+      format: document.publishing_api_document_type
+    }.deep_merge(document.format_specific_metadata.reject { |_k, v| v.blank? })
```

Comment:
@benilovj @elliotcm this is the refactoring (it has caused the previous discussion to be hidden) but the problem of clarity is still here.
However, I am leaning towards dynamic approach towards the metadata, as writing all the fields out for each specific format seemed messy. 

Also, in this system the attributes that are grouped into "format_specific_metadata" seems semantically accurate to me. I believe our app only needs to know them as a group of metadata, and there isn't specific information or behavioural difference between the specific fields inside this metadata group.